docs/image_previews.md

```diff
@@ -265,7 +265,7 @@ case "$mimetype" in
 esac
 ```
 
-`~/.config/joshuto/on_preview_removed.sh`:
```

Comment:
Is there any reason for this change?

Reply:
Both references to this script above ("Configuring Hook Scripts" and "The Hook Scripts") don't have the .sh. I blindly copy-pasted those instructions, but with the kitty modifications, but it took me a couple minutes to identify the typo.